azure-stack/operator/azure-stack-registration.md

```diff
@@ -363,17 +363,25 @@ You need to update or renew your registration in the following circumstances:
 If you want to change the subscription you use, you must first run the **Remove-AzsRegistration** cmdlet, then ensure you're signed in to the correct Azure PowerShell context. Then run **Set-AzsRegistration** with any changed parameters including `<billing model>`. While running **Remove-AzsRegistration**, you must be signed in to the subscription used during the registration and use values of the `RegistrationName` and `ResourceGroupName` parameters as shown in the administrator portal [Find current registration details](#verify-azure-stack-hub-registration):
 
   ```powershell  
+  # select the subscription used during the registration
```

Comment:
you can locate this Azure Subscription from the AzureStack Admin Portal. (specify location)

Reply:
I have added the link to find the current registration details in the last sentence of line 363.